benchmark/inference/inference_benchmark.py

```diff
@@ -121,6 +128,11 @@ def run(args: argparse.ArgumentParser) -> None:
         '--hetero-num-neighbors', default=-1, type=int,
         help='number of neighbors to sample per layer for hetero workloads')
     argparser.add_argument('--num-workers', default=2, type=int)
+    argparser.add_argument(
+        '--maybe-with-experimental', nargs='+', default=[], type=str,
```

Comment:
Why `maybe_with`? Isn't it just `experimental-options`?

Reply:
Please see the description of this argument: `(...) It may happen that it will be not possible to use some of them (warning will be produced at such a scenario)`. For example, `pytorch_scatter` experimental requires `torch>=1.12`, so it may happen that the user of this benchmark will use `--maybe-with-experimental` and it will not have any effect (in case of `torch<1.12`).

Reply:
Can't we use `action="store_true"` here, see [here](https://github.com/pyg-team/pytorch_geometric/blob/master/examples/gcn.py#L17)? Also rename to `args.experimental_node`.

Reply:
Yes, we could, but that makes sense only for an argument like `with_pytorch_scatter`. I assumed here, that there might be more experimental options in the future

Reply:
Oh, I see. I think we should not over-complicate this for now - we can revisit this once we add more experimental options. Does that work for you?

Reply:
It's ok for me.